src/include/btmem.h

```diff
@@ -567,6 +567,16 @@ struct __wt_split_page_hist {
 };
 #endif
 
+/*
+ * WT_PAGE_BLOCK_META --
+ *  Block management metadata associated with a page.
+ */
+struct __wt_page_block_meta {
+    uint64_t page_id;
```

Comment:
We are not going to introduce the reconciliation id or we are just not going to do that in the prototype?

Reply:
Just to put in writing what I mentioned in our meeting: Let's add reconciliation ID when we work on versioning pages via checkpoint IDs.